src/rate_limiter.rs

```diff
@@ -32,8 +33,11 @@ use std::time::Instant;
 const CAPACITY: u64 = 20 * 1024 * 1024;
 /// The number of bytes per second the `RateLimiter` will "leak".
 const RATE: f64 = 20.0 * 1024.0 * 1024.0;
-/// Charge (in bytes) for a client get request.
-const CLIENT_GET_CHARGE: u64 = 2 * 1024 * 1024;
+
+const GET_MUTABLE_DATA_SHELL_CHARGE: u64 = LIST_MUTABLE_DATA_PERMISSIONS_CHARGE + 88;
+const LIST_MUTABLE_DATA_PERMISSIONS_CHARGE: u64 = (500 * 44) + 40;
+const LIST_AUTH_KEYS_AND_VERSION_CHARGE: u64 = (500 * 32) + 44;
+const GET_ACCOUNT_INFO_CHARGE: u64 = 48;
```

Comment:
yeah, 22k charge for a get_account_info is bit high. but 48 means a client can sends about 20k requests or max 200k reqs (no other live client).
maybe for payload size charge, we shall have `cmp::max(payload.len(), MIN_CHARGE)`
and MIN_CHARGE can be say 1k?